chef_master/source/server_security.rst

```diff
@@ -310,6 +310,105 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Server X.Y:**  Chef Server X.Y introduces the capability to encrypt traffic between Chef Infra Server and an external PostgreSQL configuration over SSL.  These instructions are not all-encompassing and assume familiarity with PostgreSQL administration, configuration, and troubleshooting.  Consult the appropriate PostgreSQL documentation as necessary, e.g. https://www.postgresql.org/docs/9.6/ssl-tcp.html.
+
+To enable this encryption (typical scenario):
+
+1. Ensure that OpenSSL is installed on the PostgreSQL machine.
+
+2. Ensure that PostgreSQL has been built with SSL support compiled-in.
+
+3. Place SSL certificates in the proper directories on the PostgreSQL machine, and ensure correct filenames, ownerships and permissions.
+
+4. Enable SSL support on PostgreSQL, and specify paths to the SSL certificates, if necessary.
+
+   Edit the postgresql.conf file on the PostgreSQL machine and ensure the relevant entries are present and not commented out (substitute the appropriate paths):
+
+::
```

Comment:
These seem to work only if there is no indentation after the code-block. Might want to try using the following instead?
  ```
     # my code here
  ```
  5. other instructions

Reply:
See our [Style Guide](https://docs.chef.io/style_guide.html#code-blocks) for information on formatting code blocks.

Reply:
Now using `.. code-block:: bash`.  I assume that is correct?